support-macros/src/lib.rs

```diff
@@ -0,0 +1,58 @@
+#![recursion_limit = "128"]
+extern crate proc_macro;
+extern crate proc_macro2;
+
+use proc_macro::TokenStream;
+
+use quote::quote;
+use syn;
+use syn::Error;
+use syn::ItemFn;
+
+#[proc_macro_attribute]
+pub fn libtock_main(_: TokenStream, input: TokenStream) -> TokenStream {
```

Comment:
Please add some descriptive comments. My implementation is obviously incomplete, but feel free to copy from [https://github.com/jrvanwhy/libtock-rs/blob/async_main_proc_macro/proc-macros/src/lib.rs](https://github.com/jrvanwhy/libtock-rs/blob/async_main_proc_macro/proc-macros/src/lib.rs).

Reply:
Good point, done!